apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -230,7 +230,14 @@ export function PollWorkerScreen({
     setIsConfirmingEnableLiveMode(false);
   }
 
-  if (hasVotes && pollWorkerAuth.cardlessVoterUser) {
+  if (
+    hasVotes &&
+    pollWorkerAuth.cardlessVoterUser &&
+    // It's expected there are votes in app state if the state machine reports a blank page after printing.
+    // The paper was likely inserted upside down so the solution is to reload paper correctly and go back to
+    // the voting screen
```

Comment:
I don't like leaving super specific comments like this that might diverge from implementation, but I think it's important to explain why we're ok with breaking a previous constraint (that you can't enter a voting session with existing frontend ballot state)